src/ui_utils.c

```diff
@@ -298,10 +298,20 @@ static gchar *create_statusbar_statistics(GeanyDocument *doc,
 				g_string_append(stats_str, cur_tag);
 				break;
 			case 'Y':
+			{
+				gint style = sci_get_style_at(doc->editor->sci, pos);
+				gint base_style = scintilla_send_message(doc->editor->sci, SCI_GETSTYLEFROMSUBSTYLE, style, 0);
+
 				g_string_append_c(stats_str, ' ');
-				g_string_append_printf(stats_str, "%d",
-					sci_get_style_at(doc->editor->sci, pos));
+				g_string_append_printf(stats_str, "%d", base_style);
+				if (base_style != style)
+				{
+					gint sub_start = scintilla_send_message(doc->editor->sci, SCI_GETSUBSTYLESSTART, base_style, 0);
+
+					g_string_append_printf(stats_str, "+%d", style + 1 - sub_start);
```

Comment:
I changed my mind and implemented `B.S`.  Reasoning is that it looks more similar than defining the style, and has " style-and-a-half vibe" rather than an odd addition.  But that's still quite arbitrary, feel free to suggest something even better :)

Reply:
Yeah a.b is common subobject referencing syntax and here its referencing the substyle.